src/components/Calendar/ScheduleNameDialog.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { Button, Dialog, DialogTitle, DialogActions, DialogContent, TextField, MenuItem } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import { addSchedule, renameSchedule } from '../../actions/AppStoreActions';
+import { isDarkMode } from '../../helpers';
+
+const styles = () => ({
+    textField: {
+        marginBottom: '25px',
+    },
+});
+
+const ScheduleNameDialog = (props) => {
```

Comment:
Let's also de-structure here too
```suggestion
const ScheduleNameDialog = (props) => {
    const { onOpen, onClose, scheduleIndex, scheduleNames, rename } = props; // there's probably more 
```